houston/ardu/sandbox.py

```diff
@@ -195,16 +203,16 @@ def run(self, commands: Sequence[Command]) -> 'MissionOutcome':
                                        0.0, 0.0, 0.0, 0.0,
                                        -35.3632607, 149.1652351, 584)
             # 10 seconds delay to allow the robot to reach its stable state
-            delay = dronekit.Command(0, 0, 0,
-                                     3, 93, 0, 0,
-                                     10, -1, -1, -1,
-                                     0, 0, 0)
+            # delay = dronekit.Command(0, 0, 0,
+            #                         3, 93, 0, 0,
+            #                         10, -1, -1, -1,
+            #                         0, 0, 0)
 
-            cmds = [initial]
-            for cmd in commands:
-                cmds.append(cmd.to_message().to_dronekit_command())
-                cmds.append(delay)
+            cmds = [cmd.to_message().to_dronekit_command()
```

Comment:
A single-sentence comment above, explaining the conversion to dronekit, would be useful.